lib/OrganizationsFieldArray/EditOrganizationCard.js

```diff
@@ -0,0 +1,229 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { isEmpty, uniq } from 'lodash';
+import { AppIcon } from '@folio/stripes/core';
+import { FormattedMessage } from 'react-intl';
+import { Field } from 'redux-form';
+import {
+  Card,
+  IconButton,
+  Button,
+  Layout,
+  Col,
+  Label,
+  Select,
+} from '@folio/stripes/components';
+import { required } from '../validators';
+import OrganizationCard from './OrganizationCard';
+
+export default class EditOrganizationCard extends React.Component {
+  static propTypes = {
+    name: PropTypes.string,
+    index: PropTypes.string,
+    organization: PropTypes.object,
+    onMarkforDeletion: PropTypes.func,
+    onDeleteField: PropTypes.func,
+    onReplaceField: PropTypes.func,
+    roles: PropTypes.arrayOf(PropTypes.object),
+  };
+
+  renderOrgInfoHeaderEnd = () => {
+    const { name, index } = this.props;
+    return (
+      <Button
+        data-test-unlink-button
+        buttonStyle="danger"
```

Comment:
This button is misaligned vertically because of bottom margin. [Apply `marginBottom0` to it.](https://github.com/folio-org/stripes-components/tree/master/lib/Button#props)

Same thing with the "Add Organization" button when an org hasn't been selected.

Reply:
_Add Organization_ Button is rendered here https://github.com/folio-org/ui-plugin-find-organization/blob/master/OrganizationSearch/OrganizationSearch.js#L51 and looks like it doens't have a capability to render a _marginBottom0_ style prop. Might have to raise a PR against ui-plugin0find-org and add it.

Reply:
Raised a PR for this against the plugin